playbooks/roles/haproxy/templates/haproxy.cfg.j2

```diff
@@ -32,7 +39,7 @@ defaults
 frontend web
   mode http
   bind *:80
-  bind *:443 ssl crt /etc/ssl/{{ inventory_hostname }}/haproxy
+  bind *:443 ssl crt /etc/ssl/{{ inventory_hostname }}/haproxy alpn h2,http/1.1
```

Comment:
how this fix the SSL on influxdb on port 8086?
This is more valid for grafana frontend